packages/node_modules/@webex/webex-core/src/lib/services/services.js

```diff
@@ -321,7 +325,9 @@ const Services = WebexPlugin.extend({
           exists: true,
           details: 'user exists and is activated'
         },
-        this.sendUserActivation({email, reqId, token})
+        this.sendUserActivation({
+          email, reqId, token, activationOptions
```

Comment:
new line params for readability
```suggestion
          email,
          reqId,
          token,
          activationOptions
```